cmds/webboot/webboot.go

```diff
@@ -216,14 +208,54 @@ func (d *DirOption) exec(uiEvents <-chan ui.Event, menus chan<- string) (menu.En
 	return menu.PromptMenuEntry("Distros", "Choose an option:", entries, uiEvents, menus)
 }
 
+// Downloads and parses the data in distros.json to a map[string]Distro.
```

Comment:
In Go, the convention is to start the comment with the function name `distroData downloads and parses ...`